src/tests/ftest/io/ior_intercept_verify_data_integrity.py

```diff
@@ -0,0 +1,161 @@
+"""
+
+import os
+import threading
+import write_host_file
+from ior_test_base import IorTestBase
+from ior_utils import IorCommand
+from command_utils import CommandFailure
+
+
+class IorInterceptVerifyDataIntegrity(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR with mix of dfuse and
+       interception library on a multi server and multi client
+       settings and verify read/write.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        super(IorInterceptVerifyDataIntegrity, self).setUp()
+        # Following line can be removed once the constraint in the
+        # IorTestBase is resolved. #DAOS-3320
+        self.hostlist_clients = self.params.get(
+            "test_clients", "/run/hosts/*")
+        self.lock = threading.Lock()
+
+    def test_ior_intercept_verify_data(self):
+        """Jira ID: DAOS-3502.
+
+        Test Description:
+            Purpose of this test is to run ior through dfuse with
+            interception library  on 5 clients and without interception
+            library on 1 client for at least 30 minutes and verify the
+            data integrity using ior's Read Verify and Write Verify
+            options.
+
+        Use case:
+            Run ior with read, write, fpp, read verify
+            write verify for 30 minutes
+            Run ior with read, write, read verify
+            write verify for 30 minutes
+
+        :avocado: tags=all,daosio,hw,full_regression,iorinterceptverifydata
+        """
+        intercept = os.path.join(self.prefix, 'lib64', 'libioil.so')
+        with_intercept = dict()
+        self.run_multiple_ior_with_pool(with_intercept, intercept)
+
+        IorCommand.log_metrics(self.log, "5 clients - with " +
+                               "interception library", with_intercept[1])
+        IorCommand.log_metrics(self.log, "1 client - without " +
+                               "interception library", with_intercept[2])
+
+    def run_multiple_ior_with_pool(self, results, intercept=None):
+        """Execute ior with optional overrides for ior flags and object_class.
+
+        If specified the ior flags and ior daos object class parameters will
+        override the values read from the yaml file.
+
+        Args:
+            intercept (str): path to the interception library. Shall be used
+                             only for POSIX through DFUSE.
+            ior_flags (str, optional): ior flags. Defaults to None.
+            object_class (str, optional): daos object class. Defaults to None.
+        """
+        # Create a pool if one does not already exist
+        if self.pool is None:
+            self.create_pool()
+        # Update IOR params with the pool
+        self.ior_cmd.set_daos_params(self.server_group, self.pool)
+
+        # start dfuse if api is POSIX
+        if self.ior_cmd.api.value == "POSIX":
```

Comment:
Not needed, as you are only using POSIX api

Reply:
Done